modules/acs-quick-install-secured-cluster-using-helm.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_helm/install-helm-quick.adoc
+:_module-type: PROCEDURE
+[id="installing-secured-cluster-services-quickly_{context}"]
+= Installing the secured-cluster-services Helm chart without customization
+
+[role="_abstract"]
+Use the following instructions to install the secured-cluster-services Helm chart to deploy the per-cluster and per-node components (Sensor, Admission Controller, and Collector).
+
+[CAUTION]
+====
+To install Collector on systems configured with Unified Extensible Firmware Interface (UEFI) boot, you must use eBPF probes because kernel modules are unsigned, and the UEFI firmware cannot load unsigned packages.
+====
+
+.Prerequisites
+* You must have the address and the port number on which you are exposing the Central service.
+
+.Procedure
+* Run the following command:
++
+[source,terminal]
+----
+$ helm install -n stackrox --create-namespace \
+    stackrox-secured-cluster-services stackrox/secured-cluster-services \
+    -f <path_to_cluster_init_bundle.yaml> \ <1>
+    --set clusterName=<name_of_the_secured_cluster> \
+    --set centralEndpoint=<endpoint_of_Central_service> <2>
+----
+<1> Use the `-f` option to specify the path for the init bundle.
+<2> Specify Central's address and the port number. For example, `central.stackrox:443`.
```

Comment:
Not critical, but suggest avoiding the "'s" with Central (per IBM style guide). Perhaps "Specify the address and port number for Central."